tests/e2e/catalog/authorizations/test_async_authorizations.py

```
@@ -0,0 +1,55 @@
+    return async_mpt_ops.catalog.authorizations
+
+
+@pytest.fixture
+async def created_authorization(async_authorizations_service, authorization_data):
+    async with async_create_fixture_resource_and_delete(
+        async_authorizations_service, authorization_data
+    ) as authorization:
+        yield authorization
+
+
+def test_create_authorization(created_authorization, authorization_data):  # noqa: AAA01
```

Ruff (0.14.7)

Unused `noqa` directive (unknown: `AAA01`)

Remove unused `noqa` directive

(RUF100)